Source/WebCore/page/scrolling/ScrollingStateTree.cpp

```diff
@@ -347,10 +347,10 @@ void ScrollingStateTree::willRemoveNode(ScrollingStateNode& node)
     setHasChangedProperties();
 }
 
-ScrollingStateNode* ScrollingStateTree::stateNodeForID(ScrollingNodeID scrollLayerID) const
+RefPtr<ScrollingStateNode> ScrollingStateTree::stateNodeForID(ScrollingNodeID scrollLayerID) const
```

Comment:
Seems like the `scrollLayerID` argument is misnamed. It should just be `nodeID`.

Reply:
Fixed.